src/dsc.rs

```diff
@@ -98,22 +98,58 @@ pub struct Dsc {
     pub files: Vec<FileEntry>,
 }
 
+pub fn discard_pgp(dsc: &[u8]) -> &[u8] {
+    const PGP_HEADER_START: &str = "-----BEGIN PGP SIGNED MESSAGE-----\n";
+    const PGP_HEADER_END: &str = "\n\n";
+    const PGP_FOOTER_START: &str = "-----BEGIN PGP SIGNATURE-----\n";
+
+    if let Ok(dsc_str) = std::str::from_utf8(dsc) {
```

Comment:
if .dsc files always need to be utf-8 (I don't remember tbh), it might be worth just having this take a string and making the caller convert first, and then they can just raise a hard error if the file isn't valid for this.

Reply:
I've updated it to take a `&str`.